lib/game.rb

```diff
@@ -0,0 +1,139 @@
+class Game
+  include Players
+
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [[0,1,2],[3,4,5],[6,7,8],
+                      [0,3,6],[1,4,7],[2,5,8],
+                      [0,4,8],[2,4,6]]
+
+  def initialize(player_1 = Human.new("X"), player_2 = Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def self.start
+    puts "Welcome to CLI Tic Tac Toe!"
+    puts "How many players?"
+
+    num_players = gets.strip
+    case num_players
+    when "0"
+      new(Computer.new("X"), Computer.new("O"))
+    when "1"
+      puts "Are you 'X' or 'O'?"
+      human_token = gets.strip
+      if human_token == "X"
+        new(Human.new("X"), Computer.new("O"))
+      else
+        new(Computer.new("X"), Human.new("O"))
+      end
+    when "2"
+      new(Human.new("X"), Human.new("O"))
+    end
+  end
+
+  def current_player
+    if board.cells.count(player_1.token) > board.cells.count(player_2.token)
```

Comment:
Here you can use [`Integer#even?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-even-3F) or [`Integer#odd?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-odd-3F)

`@board.turn_count.even? ? @player_1 : @player_2`